### ISSUE TITLE
Songs: Lyrics not inherited from parent entry after deleting child entry's lyrics

### ISSUE THREAD

**Issue by saturclay** [VocaDB/vocadb#1674]
### Website (required)
VocaDB / UtaiteDB / [TouhouDB] (presumably true of other DBs as well)

### Describe the bug (required)
After adding and deleting the lyrics from a song, it no longer inherits the lyrics of the parent version.
See [here](https://touhoudb.com/S/83996) for the song in question.

### To Reproduce (optional)
1. Add lyrics for a song where the parent version already has lyrics and save the entry.
2. Edit the entry again and remove the lyrics.
3. Even if you revert the entry to before lyrics were added, the lyrics still don't inherit from the parent version.

### Checklist (required)

Fill out the checklist. Use ~~strikethroughs~~ or leave fields empty if you don't know or are unable to check/test.

- [x] Searched **[VocaDB/vocadb](https://github.com/VocaDB/vocadb/issues)** for duplicate issues.
- [ ] Tested on the **[beta site](https://beta.vocadb.net)**.
- [ ] Tested on different browsers (e.g. **Firefox**, **Chrome**, **Safari**, **Edge**).
- [ ] Checked in **private mode**.
- [ ] Cleared **browser cache** and **cookies**.
- [ ] Checked the **DevTools console**.
- [ ] Checked the **syslog** (mods+ only).

**Comment by andreoda** [comment]
Normally lyrics should not get removed if they already have some.
If wrong lyrics get added and there is no replacement, it should be possible to remove them at any time.
So, the parent entry lyrics should not reappear on the child entry after removed them from it (child entry).

Perhaps there could be a button on the lyrics tab to add the parent entry lyrics.

The issue is that the lyrics appear empty on children entries when in edit mode.

**Comment by saturclay** [comment]
Yeah, I think a "use parent version's lyrics" button on the child version's edit page would be good. 

I don't know how often this specific situation will come up (i.e. parent version already has correct lyrics->someone adds incorrect lyrics to child version->incorrect lyrics are removed), but I wonder if the issue would have been avoided if I had tried reverting the entry to before the wrong lyrics were added *first*, instead of removing the wrong lyrics and then trying to revert the entry when that didn't work. 🤔
